oidc.go

```diff
@@ -160,12 +178,30 @@ func (p *Provider) Endpoint() oauth2.Endpoint {
 
 // UserInfo represents the OpenID Connect userinfo claims.
 type UserInfo struct {
+	Subject     string `json:"sub"`
+	DisplayName string `json:"displayName"`
+	Email       string `json:"email"`
+	GivenName   string `json:"givenName"`
+	FamilyName  string `json:"familyName"`
+
+	claims []byte
+}
+
+type googleUserInfo struct {
 	Subject       string `json:"sub"`
-	Profile       string `json:"profile"`
+	DisplayName   string `json:"name"`
```

Comment:
Isn't that supposed to be profile in the json from Google ? Or am I confused ?

Reply:
Profile was returning an empty string.
{
    "name": "Martin Plouy",
    "given_name": "Martin",
    "family_name": "Plouy",
    "email": "martin@gipsybot.com",
    "email_verified": true,
    "locale": "en",
}

This is what I get from the google endpoint https://www.googleapis.com/oauth2/v3/userinfo (I hid the personal fields like sub)